src/image/VisitInfo.cc

```diff
@@ -360,11 +377,35 @@ namespace detail {
 int stripVisitInfoKeywords(daf::base::PropertySet& metadata) {
     int nstripped = 0;
 
-    std::vector<std::string> keyList = {"EXPID",    "EXPTIME",     "DARKTIME",     "DATE-AVG",    "TIMESYS",
-                                        "TIME-MID", "MJD-AVG-UT1", "AVG-ERA",      "BORE-RA",     "BORE-DEC",
-                                        "BORE-AZ",  "BORE-ALT",    "BORE-AIRMASS", "BORE-ROTANG", "ROTTYPE",
-                                        "OBS-LONG", "OBS-LAT",     "OBS-ELEV",     "AIRTEMP",     "AIRPRESS",
-                                        "HUMIDITY", "INSTRUMENT",  "IDNUM",        "FOCUSZ",      "OBSTYPE"};
+    std::vector<std::string> keyList = {"EXPID",
+                                        "EXPTIME",
+                                        "DARKTIME",
+                                        "DATE-AVG",
+                                        "TIMESYS",
+                                        "TIME-MID",
+                                        "MJD-AVG-UT1",
+                                        "AVG-ERA",
+                                        "BORE-RA",
+                                        "BORE-DEC",
+                                        "BORE-AZ",
+                                        "BORE-ALT",
+                                        "BORE-AIRMASS",
+                                        "BORE-ROTANG",
+                                        "ROTTYPE",
+                                        "OBS-LONG",
+                                        "OBS-LAT",
+                                        "OBS-ELEV",
+                                        "AIRTEMP",
+                                        "AIRPRESS",
+                                        "HUMIDITY",
+                                        "INSTRUMENT",
+                                        "IDNUM",
+                                        "FOCUSZ",
+                                        "OBSTYPE",
+                                        "PROGRAM",
+                                        "REASON",
+                                        "OBJECT",
+                                        "HAS-SIMULATED-CONTENT"};
```

Comment:
😬

Reply:
Yeah, that's what clang-format did, since `HAS-SIMULATED-CONTENT` is so much longer than the rest.